src/actions/CryptoExchangeActions.js

```diff
@@ -320,12 +320,32 @@ export const shiftCryptoCurrency = (swapInfo: GuiSwapInfo, onApprove: () => void
   const { account } = state.core
   dispatch({ type: 'START_SHIFT_TRANSACTION' })
 
-  const { quote, request } = swapInfo
-  const { pluginId, toNativeAmount } = quote
+  const { fromDisplayAmount, quote, request, fee, fromFiat, fromTotalFiat, toDisplayAmount, toFiat } = swapInfo
+  const { isEstimate, fromNativeAmount, toNativeAmount, networkFee, pluginId, expirationDate } = quote
   const { fromWallet, toWallet, fromCurrencyCode, toCurrencyCode } = request
   try {
     logEvent('SwapStart')
     const result: EdgeSwapResult = await quote.approve()
+
+    global.logActivity(`Swap Exchange Executed: ${account.username}`)
+    global.logActivity(`
```

Comment:
Why not pass the information as an object to the logs?

Reply:
Visual formatting would suck. Also I don't want all object info to be printed as there may be sensitive info in the object so I would have to filter out some items anyway.